examples/multisig/lib.rs

```diff
@@ -546,6 +549,7 @@ mod multisig {
                         .gas_limit(t.gas_limit)
                         .transferred_value(t.transferred_value),
                 )
+                .call_flags(CallFlags::default().set_allow_reentry(true))
```

Comment:
I think we should store this in the `Transaction` whether reentrancy should be enabled.

Reply:
hm. Indeed, as the transaction can have any other .callee aside from the mutlisig itself, this makes sense

Reply:
Yes. Let's not shake the yak here if you don't want to.